Tests/Processors/ProcessorGraphTests.cpp

```diff
@@ -0,0 +1,146 @@
+#include "gtest/gtest.h"
+#include <Processors/ProcessorGraph/ProcessorGraph.h>
+#include <modules/juce_gui_basics/juce_gui_basics.h>
+#include <Audio/AudioComponent.h>
+#include <UI/ControlPanel.h>
+
+class ProcessorGraphTest : public ::testing::Test {
+protected:
+    void SetUp() override {
+        MessageManager::deleteInstance();
+        MessageManager::getInstance();
+        AccessClass::clearAccessClassStateForTesting();
+
+        // All of these sets the global state in AccessClass in their constructors
+        audio_component = std::make_unique<AudioComponent>();
+        processor_graph = std::make_unique<ProcessorGraph>(true);
+        control_panel = std::make_unique<ControlPanel>(processor_graph.get(), audio_component.get(), true);
+    }
+
+    void TearDown() override {
+        control_panel = nullptr;
+        processor_graph = nullptr;
+        audio_component = nullptr;
+
+        AccessClass::clearAccessClassStateForTesting();
+
+        DeletedAtShutdown::deleteAll();
+        MessageManager::deleteInstance();
+    }
+
+    std::unique_ptr<ProcessorGraph> processor_graph;
+private:
+    std::unique_ptr<AudioComponent> audio_component;
+    std::unique_ptr<ControlPanel> control_panel;
+};
+
+TEST_F(ProcessorGraphTest, LoadFromXMLTest) {
+    
+    std::string default_file_reader_path = "default";
+    File currentPath = File::getSpecialLocation(File::currentApplicationFile);
+    while(currentPath.getFileName() != "plugin-GUI") {
+        currentPath = currentPath.getParentDirectory();
+    }
+    
+    currentPath = currentPath.getChildFile("Resources/FileReader/resources/structure.oebin");
```

Comment:
would this work on Windows? Since this uses forward slashes instead of backslashes.